.pre-commit-config.yaml

```diff
@@ -15,36 +15,46 @@
 default_language_version:
   python: python3.8
 
+ci:
+  autofix_prs: true
+  autoupdate_commit_msg: '[pre-commit.ci] pre-commit suggestions'
+  # submodules: true
+
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
+    rev: v3.4.0
     hooks:
-      - id: trailing-whitespace
       - id: end-of-file-fixer
+      - id: trailing-whitespace
+      #- id: check-yaml
+      - id: pretty-format-json
 
-  - repo: local
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.8.0
     hooks:
       - id: isort
         name: imports
         entry: isort
-        args: [--settings-path, ./pyproject.toml]
-        language: system
+        language: python
         types: [python]
+        require_serial: false
```

Comment:
what is `require_serial` for?

Reply:
pre-commit was complaining that it was missing

Reply:
According to this:

https://pre-commit.com/#creating-new-hooks

It's already false by default, so shouldn't be necessary